src/superannotate/lib/infrastructure/services.py

```diff
@@ -1143,7 +1150,79 @@ def get_annotations(
             )
         )
 
-    async def download_annotations(
+    async def download_big_annotation(
+        self,
+        project_id: int,
+        team_id: int,
+        folder_id: int,
+        reporter: Reporter,
+        download_path: str,
+        postfix: str,
+        item: int,
+        callback: Callable = None
+    ):
+        item_id = item['id']
+        item_name = item['name']
+        query_params = {
+            "team_id" : team_id,
+            "project_id": project_id,
+            "annotation_type": "MAIN",
+            "version":"V1.00"
+        }
+
+        url = urljoin(self.get_assets_provider_url('v1.01'), f'items/{item_id}/annotations/download')
+
+        sync_params = {
+            "team_id": team_id,
+            "project_id": project_id,
+            "desired_transform_version": "export",
+            "desired_version":"V1.00",
+            "current_transform_version": "V1.00",
```

Comment:
desired_version and current_transform_version should not be hard coded(lats hard coded as class variable) )